src/components/Book.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly avoid disabling linters and fix them because linters make developers follow best practices and write clean code.